src/vendor/components/chartJs/CustomTooltip.jsx

```diff
@@ -61,110 +61,147 @@ const styles = {
 
     const top = canvas.offsetTop + tooltipModel.caretY;
     const left = canvas.offsetLeft + tooltipModel.caretX;
     const alignments = [`x${tooltipModel.xAlign}`, `y${tooltipModel.yAlign}`];
-    // console.log(alignments);
     const inlineStyle = {
       top,
       left,
       // eslint-disable-next-line no-underscore-dangle
       fontFamily: tooltipModel._bodyFontFamily,
       // eslint-disable-next-line no-underscore-dangle
-      fontSize: `${tooltipModel._bodyFontSize}px`,
+      fontSize: `${tooltipModel.bodyFontSize}px`,
     };
-    const currPoint = tooltipModel.dataPoints[0];
-    const labelColor = {
-      backgroundColor: tooltipModel.labelColors[0].borderColor,
-    };
-    const { index } = currPoint;
-    const currSeries = series[currPoint.datasetIndex];
-
-    if (missing(currSeries)) {
-      return null;
-    }
-
-    const higherOrLower = currSeries.meta.lowerIsBetter
-      ? 'lower is better'
-      : 'higher is better';
-    const curr = currSeries.data[index];
-    const hgURL = URL({
-      path: 'https://hg.mozilla.org/mozilla-central/pushloghtml',
-      query: { changeset: curr.revision },
-    });
-    const jobURL = URL({
-      path: 'https://treeherder.mozilla.org/#/jobs',
-      query: {
-        repo: 'mozilla-central',
-        revision: curr.revision,
-        selectedJob: curr.job_id,
-        group_state: 'expanded',
-      },
-    });
+    const { datasetIndex: seriesIndex, index } = tooltipModel.dataPoints[0];
+    const { data, series } = standardOptions;
+    const currSeries = series[seriesIndex];
+    const record = data[index];
+    const HandleTooltip = standardOptions.tip;
 
     return (
       <div
         className={[classes.tooltip, ...alignments].join(' ')}
         style={inlineStyle}>
-        <div className={classes.test}>{currPoint.xLabel}</div>
-        <div>
-          <span style={labelColor} className={classes.tooltipKey} />
-          {currSeries.label}: {round(currPoint.yLabel, { places: 3 })}
-        </div>
-        <div>
-          {currPoint.yLabel} ({higherOrLower})
-        </div>
-        {(() => {
-          if (index === 0) {
-            return null;
-          }
-
-          const prev = currSeries.data[index - 1];
-          const delta = curr.value - prev.value;
-          const deltaPercentage = (delta / prev.value) * 100;
+        <HandleTooltip
+          {...{
+            record,
+            index,
+            data,
+            series: currSeries,
+            isLocked: tooltipIsLocked,
+            seri: series,
+            standardOptions,
+          }}
+        />
+      </div>
+    );
+  }
+}
+
+CustomTooltip.propTypes = {
+  classes: PropTypes.shape({}).isRequired,
+  standardOptions: PropTypes.shape({
+    tip: PropTypes.func.isRequired,
+    series: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  }).isRequired,
+  tooltipModel: PropTypes.shape({}),
+  tooltipIsLocked: PropTypes.bool.isRequired,
+  canvas: PropTypes.shape({}),
+};
```

Comment:
The above looks like a lot of change, but I simply split out the specific tip from the tip layout:  This layout is fantastic work done a contributor which I do not what to mess with.